src/edu/ucsd/library/xdre/web/CollectionOperationController.java

```diff
@@ -887,6 +890,17 @@ else if(ds.equals(dsDest) || !dsDest.startsWith("ts/"))
 								  }
 							  
 								  objTitle = StringUtils.isEmpty(objTitle) ? "[Object]" : objTitle;
+
+								  if (source.equalsIgnoreCase("bib")) {
+									  // Roger record: Test to report conversion errors
+									  try {
+										  testConversion(record.toRDFXML());
```

Comment:
What are your thoughts about a more explicit name for this method? It seems to only be relevant, as far as I can tell, for catalog records. Should we indicate something to that extent in the method name?

Reply:
It sounds good to use a more explicit name. How about `testRogerRecordConversion`?